### ISSUE TITLE
Accessibility: Side Bar becomes Non-functional at 320X256 - Violates 1.4.10 Reflow WCAG 2.1 Requirement

### ISSUE THREAD

**Issue by anitakumari28** [microsoft/vscode#249951]
- VS Code Version: 1.100.0
- OS Version: macOS Sequoia 15.4.1

Steps to Reproduce:

1. Launch the VSCode and then resize the window to 320X256 viewport using Toggle Developer Tools
2. Try interacting with the side bar and resizing it using the handle as given in the references.

When resizing the VSCode to a viewport of 320X256, the Side Bar functionality become non-operational. This includes the inability to resize the panel using the side bar resizer handle. This issue violates WCAG 2.1 Success Criteria 1.4.10: Reflow ([Understanding 1.4.10 Reflow](https://www.w3.org/WAI/WCAG22/UNDERSTANDING/reflow.html)), as essential functionality is lost at small viewport.

References:
1. 
<img width="280" alt="Image" src="https://github.com/user-attachments/assets/3b2cb456-83fe-4e28-82d6-f83ef32baa08" />

2. https://github.com/user-attachments/assets/690ab38c-16c8-44fa-827c-b010aef3e81d

**Comment by anitakumari28** [comment]
I tried with the latest version of VSCode (i.e. 1.100.2), but the issue still persists.

**Comment by TylerLeonhardt** [comment]
@benibenj @joaomoreno I feel like one of you own this?

**Comment by joaomoreno** [comment]
Even as the window is at its minimum size, the side bar is still functional and allows the user to hide it.

https://github.com/user-attachments/assets/06e0e9fc-78db-4db7-bfdd-5e0fac12c1ca
